### ISSUE TITLE
Database access session recording and viewing

### ISSUE THREAD

**Issue by r0mant** [gravitational/teleport#5799]
### What

Record database sessions and allow to replay them similar to SSH.

**Comment by stevenGravy** [comment]
We have been seeing a number of requests for this.

**Comment by kenske** [comment]
What is currently being stored in the tar files that are uploaded after a DB session is finished?

**Comment by greedy52** [comment]
Tasks
- [x] RFD
- [x] Play Postgres sessions

Other related issues:
- [ ] https://github.com/gravitational/teleport/issues/36972
- [x] https://github.com/gravitational/teleport/issues/26521

**Comment by philip-teleport** [comment]
+1 for MySQL session recording from a prospect

**Comment by philip-teleport** [comment]
+1 from a prospect looking for database session recording to be supported for more databases, particularly all of the supported AWS databases. 
This would make their audit reviews far easier, where they can show database session playbacks to auditors when they ask for details of a specific session, rather running reports in their SIEM based on Audit Log events to gather this information.

**Comment by greedy52** [comment]
> +1 from a prospect looking for database session recording to be supported for more databases, particularly all of the supported AWS databases. This would make their audit reviews far easier, where they can show database session playbacks to auditors when they ask for details of a specific session, rather running reports in their SIEM based on Audit Log events to gather this information.

Database recording is always saved in backend, in `json` format, in addition to audit events. you can playback them with `tsh play <session-id> --format json`. This ticket is for "interactive" playback.

**Comment by philip-teleport** [comment]
@greedy52 thanks for the clarification. The [playback](https://goteleport.com/docs/reference/architecture/session-recording/#playback) section of this guide implies only SSH and Kubernetes sessions are supported by `tsh play`. Should I raise a docs issue for this?

**Comment by greedy52** [comment]
> [@greedy52](https://github.com/greedy52) thanks for the clarification. The [playback](https://goteleport.com/docs/reference/architecture/session-recording/#playback) section of this guide implies only SSH and Kubernetes sessions are supported by `tsh play`. Should I raise a docs issue for this?

good catch! we have an existing issue on this topic https://github.com/gravitational/teleport/issues/36972 but it is not doc related. If would be great if you can file a separate doc issue 🙏

**Comment by pschisa** [comment]
+1 for mysql session recording

**Comment by noam-zimet-teleport** [comment]
Another +1 for mysql session recording
